langkit/templates/python_api/playground_py.mako

```diff
@@ -14,7 +14,6 @@ from IPython.terminal.ipapp import load_default_config
 import ${module_name}
 import ${module_name} as ${short_name}
 
-
```

Comment:
We keep two empty lines between top-level imports and the rest of our Python scripts: can you add this one back? (no worry, I’ll update the style checker)